hermes/src/api/types.rs

```diff
@@ -222,7 +228,7 @@ pub struct BinaryPriceUpdate {
 
 #[derive(Debug, Clone, serde::Serialize, serde::Deserialize, ToSchema)]
 pub struct ParsedPriceUpdate {
-    pub id:        PriceIdentifier,
+    pub id:        RpcPriceIdentifier,
```

Comment:
convert this to `RpcPriceIdentifier` because `PriceIdentifier` is from `pyth-sdk-rs` and doesn't have `ToSchema` trait